### ISSUE TITLE
provide a snap app

### ISSUE THREAD

**Issue by gocarlos** [JPEWdev/icecream-sundae#2]
this software could be packaged as a snap to be able to distribute it under linux easily

**Comment by gocarlos** [comment]
[![Get it from the Snap Store](https://snapcraft.io/static/images/badges/en/snap-store-black.svg)](https://snapcraft.io/icecream-sundae-snap)


done

**Comment by JPEWdev** [comment]
Cool. I don't know anything about snaps, will this automatically update when a new version is released?

**Comment by gocarlos** [comment]
normally yes, snaps has the notion of channels, -> master is automatically pushed to edge channel, you then can promote a build to `candidate` and then to `stable` -> app on users machine is then automatically updated

right now I did this in my repo:  https://github.com/gocarlos/icecream-sundae-snap and took the name `icecream-sundae-snap` so the name `icecream-sundae` should still be free...

you can also get my snapcraft.yaml file from the repo above and release a own version with the official name, should not take more then 5 min: 

https://snapcraft.io/build
you can connect github automatically, so not even needed to build the release yourself...

let me know what you think.

**Comment by JPEWdev** [comment]
Hmm, OK. I got it setup (https://github.com/JPEWdev/icecream-sundae/commit/cd6f260e33c4caf1be07c80532f8b5b820856e49), but using the automated build service on snapcraft.io doesn't appear to work. It looks like icecream-sundae use too new a version of meson (https://build.snapcraft.io/user/JPEWdev/icecream-sundae) I added "base: core18" to the snapcraft.yml file, but it doesn't appear to have any effect when building on snapcraft.io.

Any ideas?
